test/Governance.ts

```diff
@@ -1194,6 +1196,135 @@ describe("Governance test", function () {
     });
 });
 
+describe("Votesbook test", function () {
```

Comment:
These are not Votesbook test anymore - it suppose there is no votebook anymore. Rename it. For example to "Governance voting".